static/app/views/performance/transactionSummary/transactionOverview/trendChart/index.tsx

```diff
@@ -116,31 +135,124 @@ function TrendChart({
   const trendDisplay = generateTrendFunctionAsString(trendFunction, trendParameter);
 
+  function getSelectedTransaction(
+    trendChangeType: TrendChangeType,
+    transactions?: NormalizedTrendsTransaction[]
+  ): NormalizedTrendsTransaction | undefined {
+    const queryKey = getSelectedQueryKey(trendChangeType);
+    const selectedTransactionName = decodeScalar(location.query[queryKey]);
+
+    if (!transactions) {
+      return undefined;
+    }
+
+    const selectedTransaction = transactions.find(
+      transaction =>
+        `${transaction.transaction}-${transaction.project}` === selectedTransactionName
+    );
+
+    if (selectedTransaction) {
+      return selectedTransaction;
+    }
+
+    return transactions.length > 0 ? transactions[0] : undefined;
+  }
+
+  const trendView = eventView.clone() as TrendView;
+  modifyTrendView(
+    trendView,
+    location,
+    TrendChangeType.REGRESSION,
+    projects,
+    organization as Organization
+  );
+
   return (
     <Fragment>
       {header}
-      <EventsRequest
-        {...requestCommonProps}
-        organization={organization}
-        showLoading={false}
-        includePrevious={false}
-        yAxis={trendDisplay}
-        currentSeriesNames={[trendDisplay]}
-        partial
-        withoutZerofill={withoutZerofill}
-        referrer="api.performance.transaction-summary.trends-chart"
-      >
-        {({errored, loading, reloading, timeseriesData, timeframe: timeFrame}) => (
-          <Content
-            series={timeseriesData}
-            errored={errored}
-            loading={loading}
-            reloading={reloading}
-            timeFrame={timeFrame}
-            {...contentCommonProps}
-          />
-        )}
-      </EventsRequest>
+      {withBreakpoint ? (
+        // queries events-trends-statsv2 for breakpoint data
+        <TrendsDiscoverQuery
+          eventView={trendView}
+          orgSlug={organization.slug}
+          location={location}
+          limit={1}
+          withBreakpoint={withBreakpoint}
+        >
+          {({trendsData}) => {
+            const events = normalizeTrends(
+              (trendsData && trendsData.events && trendsData.events.data) || []
+            );
+
+            // keep trend change type as regression until the backend can support passing the type
+            const selectedTransaction = getSelectedTransaction(
+              TrendChangeType.REGRESSION,
+              events
+            );
+
+            return (
+              // queries events-stats for trend data
+              <EventsRequest
+                {...requestCommonProps}
+                organization={organization}
+                showLoading={false}
+                includePrevious={false}
+                yAxis={trendDisplay}
+                currentSeriesNames={[trendDisplay]}
+                partial
+                withoutZerofill={withoutZerofill}
+                referrer="api.performance.transaction-summary.trends-chart"
+              >
```

Comment:
events-trends-statsv2 returns timeseries data in `stats` property. you might not need to fetch timeseries data separately here

Reply:
ah right, if there are no detected regressions `stats` is empty. it's not ideal to make a second query just to get the timeseries. we can keep the second call for now. I'll be making changes to the backend so that the stats data will be always returned